test/options/generatereadmemd-azure-true/Expected/AcceptanceTests/ParameterFlattening/README.md

```diff
@@ -1,3 +1,8 @@
+---
+uid: azure-arm-parameterflattening
+summary: *content
+---
+
```

Comment:
@daschult can we make sure that there is an empty line after summary, and no empty line after the second `---`?

Reply:
Sure. Just out of curiosity, why does the whitespace matter? Is it style/preference, or does your code require it?

Reply:
@daschult I've noticed that without the whitespace sometimes the system fails to pick up the package information.

Reply:
Okay, done. Please take another look.